src/pages/VenueEntrancePage/VenueEntrancePage.tsx

```diff
@@ -42,14 +42,6 @@ export const VenueEntrancePage: React.FunctionComponent<{}> = () => {
     !venue.entrance.length ||
     venue.entrance.length < parseInt(step)
   ) {
-    const enteredVenueIds = profile.enteredVenueIds ?? [];
```

Comment:
is this being removed because we've added it to `VenuePage`?

Reply:
Looks like it. And by being there, it should mean that it counts (if unique) no matter what venue URL we end up on.

Reply:
Yup, the venue page is now responsible for this logic.